jekyll/_layouts/classic-docs.html

```diff
@@ -48,6 +48,16 @@
         {% endif %}
       </article>
     </div><!-- .article-container -->
+
+      {% if page.toc != false %}
```

Comment:
Any reason not to simplify to this?

```suggestion
      {% if page.toc %}
```

Reply:
I found that the scoping from the toc-readme:
```
  # Enable TOC by default
  - scope:
      path: ""
    values:
      toc: true
```
wasn't working at setting the toc default value, so I kept the inverted check 🤷🏾